src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/TransactionOraclesTest.java

```diff
@@ -132,22 +130,35 @@ public void dOracleRespondTest(TestContext context) {
         context,
         t -> {
           try {
+            OracleQueriesResult oracleQueriesResult =
+                this.aeternityServiceNative.oracles.blockingGetOracleQueries(
+                    oracleAccount.getPublicKey().replace("ak_", "ok_"),
```

Comment:
a discussed refactor and extract to BaseKeyPair or Util class

Reply:
added methods in the BaseKeyPair class